api/server/error_response.go

```diff
@@ -28,6 +28,11 @@ func simpleError(err error) *models.Error {
 // TODO delete me !
 func handleV1ErrorResponse(ctx *gin.Context, err error) {
 	log := common.Logger(ctx)
+	if ctx.Err() == context.Canceled {
```

Comment:
I don't think so. The contract for an http request context on the server side is that it will be closed when the client closes the connection.

```
// For incoming server requests, the context is canceled when the
  // client's connection closes, the request is canceled (with HTTP/2),
  // or when the ServeHTTP method returns
```

Reply:
I've updated the check to use ctx.Request.Context() instead of the gin.Context, so we should be good now.